readthedocs/config/config.py

```diff
@@ -620,7 +621,7 @@ def conda(self):
     @property
     def build(self):
         """The docker image used by the builders."""
-        return Build(**self._config['build'])
+        return OldBuild(**self._config['build'])
```

Comment:
Can we think a little more about these names to reflect the support of `build.languages` (or `build.tools`)? I think that old/new is not explicit enough to realize what are the differences between each other.

Ideas (none of them are good ones, tho): `BuildWithTools`, `BuildAsdf`, `BuildRuntimeTools`

Reply:
I think BuildWithTools is fine. And I'll add a helper method, so we don't need to check with isinstance.